render/detailed_node_test.go

```diff
@@ -39,14 +43,14 @@ func TestOriginTable(t *testing.T) {
 		serverHostNodeID: {
 			Title:   "Origin Host",
 			Numeric: false,
-			Rows: []report.Row{
-				{"Host name", serverHostName, ""},
+			Rows: []Row{
+				{"Host name", serverHostID, ""},
```

Comment:
This isn't semantically correct. We currently set host ID == hostname in the probe, but they're not the same concept.